packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -63,6 +63,12 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
+  <PropertyGroup>
+    <devEnabled>true</devEnabled>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
Ideally it should be a property that we could override from the VS property sheet UI. It may be a subject for another PR though.  The experience must be similar to how we set the C++/WinRT properties. I did not do it myself, so I do not know all details. This articles seems to have some info about it: https://www.codeproject.com/articles/881782/creating-custom-ui-property-pages-sheets-in-visual

Reply:
Lets file an issue to get the proper property sheet UI, so we dont forget about it as an idea.

Reply:
opened another issue to follow up on this
https://github.com/microsoft/react-native-windows/issues/4280